pom.xml

```diff
@@ -110,6 +110,16 @@ under the License.
         <!-- Can be set to any value to reproduce a specific build. -->
         <test.randomization.seed/>
         <test.unit.pattern>**/*Test.*</test.unit.pattern>
+
```

Comment:
I guess there is something wrong during rebase/merge. L#85~L#101 should hold the very first place.